unity-client/Assets/ABConverter/Core.cs

```diff
@@ -106,6 +111,44 @@ void UpdateLoop()
 
                         env.assetDatabase.Refresh();
 
+                        if (assetsAlreadyDumped && DataStore.i.ABConversorGLTFMissingDependencies.Count > 0 && sceneCid != null)
```

Comment:
Please we can move the `DataStore` variables to a `ABConverter` local storage class? We should aim to have the `ABConverter` as decoupled as possible for the rest of our codebase, as it should deserve its own repo in the future. (the only reason we can't do this right now is the GLTF importer dependency).

Reply:
The refactor to detect dependencies first and add them to the rawContents to be downloaded made this change unneeded so I reverted it.